File: PlayMath.API/models/Article.cs
using System;
using System.Collections.Generic;

namespace PlayMath.API.Models
{
    public class Article
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string SubTitle { get; set; }
        public string Body { get; set; }
        public DateTime Published { get; set; }
        public User Writer { get; set; }
        public bool IsApproved { get; set; }
        public bool IsDeleted { get; set; }
        public ArticleCategory Category { get; set; }

    }
}

File: PlayMath-SPA/src/app/_models/Article.ts
import { Category } from './Category';
import { User } from './User';

export interface Article {
    id: number;
    title: string;
    subTitle: string;
    category: Category;
    body: string;

    writer: User;
    writerId: string;
    writerName: string;
    published: Date;
}
